frame/uniques/src/tests.rs

```diff
@@ -71,6 +70,18 @@ fn attributes(collection: u32) -> Vec<(Option<u32>, Vec<u8>, Vec<u8>)> {
 	s
 }
 
+fn events() -> Vec<Event<Test>> {
```

Comment:
I keep seeing this pattern in every pallet :see_no_evil:  
Really makes me want to write a generic function and refactor them.

Reply:
It would be nice to have a more advanced function to test the events, for example, to check that some event is within the array, but has different params. That would help to understand whether the params are wrong or the event wasn't emitted

Reply:
do it.